src/test/kotlin/com/terraformation/backend/accelerator/ParticipantProjectSpeciesServiceTest.kt

```diff
@@ -177,6 +197,11 @@ class ParticipantProjectSpeciesServiceTest : DatabaseTest(), RunsAsUser {
             ParticipantProjectSpeciesAddedEvent(
                 deliverableId = deliverableId, participantProjectSpecies = it)
           })
+
+      io.mockk.verify(exactly = 1) {
```

Comment:
We normally use static imports for MockK functions, so this would be `verify(exactly=1)` here.

Reply:
Also, there could be a comment here that this is to test a performance optimization, rather than to test correctness; if we introduce some form of application-wide caching at some point, this might start failing even though the end result is correct, and a comment will make it easy for someone to adjust/remove this check as needed.